### PR TITLE
feature/dashboard

### PR DESCRIPTION
**Pull request by haritonasty**
**Summary**
1) expanded width for `.page`
2) changed `Featured insights` to `Latest Insights`
3) Removed some legacy and unused code from `App.scss` and `variables.scss`

image

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -1,96 +1,89 @@
 @import 'variables.scss';
 @import '~@santiment-network/ui/variables.scss';
+@import './mixins.scss';
```
Comment (DmitriVanGuard) [inline src/App.scss]: `'~@santiment-network/ui/mixins.scss'` could be used instead

File: src/components/Insight/InsightsScrollable.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <Panel className={styles.insights__panel}>
+      <div className={styles.insights__list}>
+        {type === TYPES.featured && (
+          <InsightsFeatured
+            maxLines={maxLines}
+            multilineTextId={multilineTextId}
+            className={styles.insights__card}
+          />
+        )}
+        {type === TYPES.latest && (
```
Comment (DmitriVanGuard) [inline src/components/Insight/InsightsScrollable.js]: This type of `if statements` (LOC `17`,`24`) could be refactored with a dictionary data structure  
```js
const El = TYPES[type];
<El maxLines=....... />
```

File: src/components/Insight/InsightsScrollable.module.scss [hunk 3]
```diff
@@ -1,5 +1,5 @@
 @import '~@santiment-network/ui/variables.scss';
-@import '../../mixins.scss';
+@import '../../mixins';
```
Comment (DmitriVanGuard) [inline src/components/Insight/InsightsScrollable.module.scss]: Is it possible here to use `san-ui` mixins ?

File: src/components/Navbar/Navbar.module.scss [hunk 4]
```diff
@@ -2,11 +2,11 @@
 @import './../../mixins.scss';
 
 .header {
-  margin: 0 !important;
-  margin-bottom: 2rem !important;
+  margin: 0;
+  margin-bottom: 32px;
```
Comment (DmitriVanGuard) [inline src/components/Navbar/Navbar.module.scss]: Could be written as a shorthand: `margin: 0 0 32px;`

File: src/components/Insight/InsightsScrollable.module.scss (hunk 3 above)
Comment (haritonasty) [reply src/components/Insight/InsightsScrollable.module.scss]: of course)
missed that moment, when in `san-ui` appeared `responsive` mixin (:

**Pull request merged**